decidim-proposals/lib/decidim/proposals.rb

```diff
@@ -45,6 +45,11 @@ module Proposals
     config_accessor :process_group_highlighted_proposals_limit do
       3
     end
+
+    # Public Setting that defines after how many days a proposal is overdue
+    config_accessor :process_group_highlighted_proposals_overdue do
```

Comment:
```suggestion
    config_accessor :unanswered_proposals_overdue do
```
I suggest this name better